.github/ISSUE_TEMPLATE/release-todo-issue-template.yml

```
@@ -126,8 +114,8 @@ body:
       **_<details closed><summary>Release Execution</summary>_**
 
       The release is performed by running one of the release workflows below:
-      - Preview Release
-      - Production Release
+      - [Preview Release](https://github.com/KinsonDigital/Velaptor/actions/workflows/preview-release.yml)
+      - [Production Release](https://github.com/KinsonDigital/Velaptor/actions/workflows/qa-prod-releases.yml)
```

Please fix URL's. They used to have the TODO instead of the project name, but when you replaced the entire template with Velaptor's, you forgot to fix the URL's to show the todo items

```suggestion
      - [Preview Release](https://github.com/KinsonDigital/[TODO: add project name here]/actions/workflows/preview-release.yml)
      - [Production Release](https://github.com/KinsonDigital/[TODO: add project name here]/actions/workflows/qa-prod-releases.yml)
```